src/libmongoc/tests/unified/runner.c

```diff
@@ -26,6 +26,86 @@
 #include "util.h"
 
 
+typedef struct {
+   const char *file_description;
+   const char *test_description;
+} skipped_unified_test_t;
+
+#define SKIP_ALL_TESTS NULL
```

Comment:
While I could have used this for `unacknowledged-bulkWrite-delete-hint-clientError` and related tests, I decided not to as we know exactly what tests should be skipped there. This was really only necessary for cases where we can't even parse tests from a file due to an unsupported schema version.

Reply:
Having one place to list explicitly skipped tests, with an option to skip all tests in one file is helpful.